site/docs/patterns/header-block.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Header block
+layout: DetailPattern
+tags:
+  - pattern
+aliases:
+  - /salt/patterns/header-block
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-header-block--header-block", label: "Header block examples" },
+    ]
+  components: ["Dialog", "Drawer", "Overlay"]
+---
+
+Header block is a standardized header for container components and application regions, used to display various text elements depending on scenarios. Headings can also display accent and status to highlight messaging.
+
+<Diagram
+  src="/img/patterns/header-block/hero-shot.png"
+  alt="Example header block. Displaying close action, status, preheader and description."
+  border
+/>
+
+## When to use
+
+Use this pattern to:
```

Comment:
This pattern is very similar to https://www.saltdesignsystem.com/salt/patterns/list-filtering, but seems documented slightly differently. Do we need to align?